recipes/primecountpy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "primecountpy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/primecountpy-{{ version }}.tar.gz
+  sha256: 78fe7cc32115f0669a45d7c90faaf39f7ce3939e39e2e7e5f14c17fe4bff0676
+
+build:
+  skip: true  # [py<37]
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+  host:
+    - cysignals
+    - cython
+    - pip
+    - primecount 7.2
```

Comment:
You should add this pinning to:
https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml